<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TwitterPage</class>
 <widget class="QWidget" name="TwitterPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TwitterPage</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>929</width>
     <height>579</height>
    </rect>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>931</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QListWidget" name="TwitList_">
       <property name="autoScroll">
        <bool>false</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="gridSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLineEdit" name="TwitEdit_"/>
       </item>
       <item>
        <widget class="QPushButton" name="TwitButton_">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
